windows-packaging/CalicoWindows/cni.conf.template

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Calico",
+  "windows_use_single_network": true,
+
+  "cniVersion": "0.3.0",
```

Comment:
Looks like maybe we can use `0.3.1` : https://github.com/tigera/operator/blob/release-v1.5/pkg/render/node.go#L333